internal/batch/batch_processor.go

```diff
@@ -403,7 +412,7 @@ func (bp *batchProcessor) initPayload(id *fftypes.UUID, flushWork []*batchWork)
 	return payload
 }
 
-func (bp *batchProcessor) getNextNonce(ctx context.Context, state *dispatchState, nonceKeyHash *fftypes.Bytes32, contextHash *fftypes.Bytes32) (int64, error) {
+func (bm *batchManager) getNextNonce(ctx context.Context, state *dispatchState, nonceKeyHash *fftypes.Bytes32, contextHash *fftypes.Bytes32) (int64, error) {
```

Comment:
I had to reparent a few methods from batchProcessor to batchManager in order to wire up the LoadContexts functionality externally... didn't move the methods to the other file in the interest of keeping the diff smaller, but I could move them if it seems confusing now.

Reply:
I think moving it would be better if that's ok @awrichar